### PR TITLE
feature-5232-Security-Scans-OSV-Scanner

### PR DESCRIPTION
**Pull request by btme0011**
## What does this PR do?

Added OSV-Scanner while pushing and raising pull_request

## Test Plan

1) Success test - 
Screenshot 2023-10-05 at 4 10 20 PM

2) Failure test - 
Screenshot 2023-10-05 at 3 51 20 PM

## Related PRs and Issues

- #5232 

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: .github/workflows/osv-scanner.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          go-version: '1.19'
+
+      - name: Install OSV Scanner
+        run: |
+          go install github.com/google/osv-scanner/cmd/osv-scanner@latest
+
+      - name: Scan for Vulnerabilities
+        run: |
+          osv-scanner -r .
```
Comment (stnguyen90) [inline .github/workflows/osv-scanner.yml:27]: Let's use this github action/workflow instead: https://google.github.io/osv-scanner/github-action/

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: .github/workflows/osv-scanner.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: OSV Scanner
```
Comment (stnguyen90) [inline .github/workflows/osv-scanner.yml:1]: Let's change this to check-dependencies.yml and the name to Check dependencies

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by gewenyu99** [comment]
Hey there! There were a lot of big PRs during this Hacktoberfest, and we wanted to give everyone ample time to collaborate with our engineering team. If you were able to merge your PRs during October, amazing. If it’s still not merged, don’t worry about it either. Either way, we’ve got your Hacktoberfest swag minted and ready to ship. 

Please comment with your Discord username here so we can contact you about your shipping information to deliver your Hacktoberfest swag.

**Pull request merged**